jujugui/static/gui/src/app/maraca/maraca.js

```diff
@@ -2,7 +2,11 @@
 'use strict';
 
 const clonedeep = require('lodash.clonedeep');
-const deepmerge = require('deepmerge');
+let deepmerge = require('deepmerge');
+// Handle differences between how the tests load this code.
```

Comment:
Can you expand on this? I have seen these issues before when not using the `import` syntax with webpack but can you elaborate in this comment for future us.

Reply:
Now that we're using webpack should we switch to the `import` syntax? Can we `import` the sass?